gui/control_main.py

```diff
@@ -5202,24 +5165,41 @@ def controlEnabled(self):
             and self.controlMasterCheckBox.isChecked()
         )
 
-    def send_to_server(self, s):
-        if s == "lockControl":
+    def send_to_server(self, function_name: str, args: "Optional[List]" = None, kwargs: "Optional[Dict]" = None):
```

Comment:
Can you explain why you incorporated this function as part of the class? There isn't anything inherently specific to the class, so I would have left it as a utility function.

Reply:
realized that the generate_server_message() would only be called within send_to_server() now, so would only be required as part of ControlMain(), and confirmed by @vshekar ... I think it makes sense as it is.